scripts/update-drafts.ts

```diff
@@ -50,17 +55,20 @@ async function main() {
   // Iterate BCD and group compat features by spec.
   const specToCompatFeatures = new Map<object, Set<string>>();
   for (const feature of compat.walk()) {
-    // Skip any BCD keys already used in web-features.
-    if (usedFeatures.has(feature.id)) {
+
+    // Skip deprecated and non-standard features.
+    const status = feature.data.__compat.status;
```

Comment:
I did get this code from some other PR because the check before was skipping all of the features, so would it be ok to delete all this check knowing that it will be added afterwards?

Reply:
I wasn't sure what would happen without trying it myself, so I tried simplifying this again like in https://github.com/web-platform-dx/web-features/pull/1292 and confirmed it produces the same resulting draft features written to disk. I'll commit that change.